barebox/config

```diff
@@ -312,7 +312,7 @@ CONFIG_CMD_DRVINFO=y
 CONFIG_CMD_HELP=y
 CONFIG_LONGHELP=y
 CONFIG_CMD_IOMEM=y
-CONFIG_CMD_IMD=y
+# CONFIG_CMD_IMD is not set
```

Comment:
I think that change shouldn't be necessary any more. Building the command should be fixed with the patch on our barebox branch (that you pulled in already).

Reply:
Ah, good point! I'll remove it